components/about_build_modal/about_build_modal.jsx

```diff
@@ -219,7 +219,7 @@ export default class AboutBuildModal extends React.PureComponent {
                         <p>
                             <FormattedHTMLMessage
                                 id='about.notice'
-                                defaultMessage='Mattermost is made possible by the open source software used in our <a href="https://about.mattermost.com/platform-notice-txt/" target="_blank">platform</a>, <a href="https://about.mattermost.com/desktop-notice-txt/" target="_blank">desktop</a> and <a href="https://about.mattermost.com/mobile-notice-txt/" target="_blank">mobile</a> apps.'
+                                defaultMessage='Mattermost is made possible by the open source software used in our <a href="https://about.mattermost.com/platform-notice-txt/" target="_blank">server</a>, <a href="https://about.mattermost.com/desktop-notice-txt/" target="_blank">desktop</a> and <a href="https://about.mattermost.com/mobile-notice-txt/" target="_blank">mobile</a> apps.'
```

Comment:
What if we change this to markdown?

Reply:
Oh right, thanks for the reminder.  I forgot about that.  Now updated :)